### PR TITLE
New admin navbar

### PR DESCRIPTION
**Pull request by Robertgervais**

### CODE REVIEW

**Comment by Robertgervais** [comment]
nav

File: app/assets/stylesheets/new_admin.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+img {
```
Comment (sethherr) [inline app/assets/stylesheets/new_admin.css]: When writing css, you need to scope the css so that it only applies to a specific thing.

File: app/views/admin/feedbacks/index.html.haml [hunk 2]
```diff
@@ -53,16 +53,16 @@
             = feedback.name
           %td.large-screens
             = feedback.body.truncate(100)
-        
+
 .paginate-container
   = paginate @feedbacks
 
-= javascript_include_tag "//www.google.com/jsapi", "chartkick" 
+= javascript_include_tag "//www.google.com/jsapi", "chartkick"
```
Comment (sethherr) [inline app/views/admin/feedbacks/index.html.haml]: Why do you include this?

File: config/routes.rb [hunk 3]
```diff
@@ -28,7 +28,9 @@
     get slug, to: 'landing_pages#show', organization_id: slug
   end
 
+
```
Comment (sethherr) [inline config/routes.rb]: Don't need to add this whitespace, probably

File: app/views/layouts/new_admin.html.haml [hunk 4]
```diff
@@ -6,60 +6,52 @@
...
+          %li.nav-item.px-2
             = admin_nav_link "Paid Features", admin_paid_features_path
-          %li
+          %li.nav-item.px-2
             = admin_nav_link "Payments", admin_payments_path
-          %li
+          %li.nav-item.px-2
             = admin_nav_link "Invoices", admin_invoices_path
 
-          - if Rails.env.development?
```
Comment (sethherr) [inline app/views/layouts/new_admin.html.haml]: Why did you remove this section?
